tools/ldb_cmd.cc

```diff
@@ -3444,88 +3444,101 @@ void DBLiveFilesMetadataDumperCommand::DoCommand() {
   }
   Status s;
 
-  std::cout << "Live SST Files:" << std::endl;
-  std::vector<LiveFileMetaData> metadata;
-  db_->GetLiveFilesMetaData(&metadata);
+  std::cout << "Live SST and Blob Files:" << std::endl;
+  std::vector<ColumnFamilyMetaData> metadata;
+  db_->GetAllColumnFamilyMetaData(&metadata);
   if (sort_by_filename_) {
     // Sort metadata vector by filename.
-    std::sort(metadata.begin(), metadata.end(),
-              [](const LiveFileMetaData& a, const LiveFileMetaData& b) -> bool {
-                std::string aName = a.db_path + a.name;
-                std::string bName = b.db_path + b.name;
-                return (aName.compare(bName) < 0);
-              });
-    for (auto& fileMetadata : metadata) {
-      // The fileMetada.name alwasy starts with "/",
-      // however fileMetada.db_path is the string provided by
-      // the user as an input. Therefore we check if we can
-      // concantenate the two string sdirectly or if we need to
-      // drop a possible extra "/" at the end of fileMetadata.db_path.
-      std::string filename = fileMetadata.db_path + "/" + fileMetadata.name;
-      // Drops any repeating '/' character that could happen during
-      // concatenation of db path and file name.
-      filename = NormalizePath(filename);
-      std::string cf = fileMetadata.column_family_name;
-      int level = fileMetadata.level;
-      std::cout << filename << " : level " << level << ", column family '" << cf
-                << "'" << std::endl;
+    std::map<std::string, std::string> filenametoOutputStr;
+    for (auto& columnMetadata : metadata) {
```

Comment:
RocksDB follows the Google C++ Style Guide, which typically uses `snake_case` for variable names. Minor and I realize the pre-existing code is in violation but would be nice to fix.